docs/source/contributing/unit_testing.md

```diff
@@ -0,0 +1,38 @@
+# Testing
+
+benchmark-wrapper uses [tox](https://pypi.org/project/tox/) and [pytest](https://docs.pytest.org/en/6.2.x/) for unit testing as well as documentation build testing.
+
+As a quick reminder, unit testing is defined as follows by [STF](https://softwaretestingfundamentals.com/unit-testing/):
+
+> UNIT TESTING, also known as COMPONENT TESTING,  is a level of software testing where individual units / components of a software are tested. The purpose is to validate that each unit of the software performs as designed.
```

Comment:
extremely minor nit but there's 2 spaces after the second ,
i.e.
COMPONENT TESTING,  is....

Reply:
Woah nice catch. That's funny as that's a copy-paste from https://softwaretestingfundamentals.com/unit-testing/. I removed the space though and will commit and push